front/src/routes/integration/all/nuki/discover-http/index.js

```diff
@@ -0,0 +1,39 @@
+import { Component } from 'preact';
+import { connect } from 'unistore/preact';
+import actions from '../actions';
+import NukiPage from '../NukiPage';
+import DiscoverTab from './DiscoverTab';
+import { WEBSOCKET_MESSAGE_TYPES } from '../../../../../../../server/utils/constants';
+
+class NukiIntegration extends Component {
+  async componentWillMount() {
+    this.props.getDiscoveredNukiDevices('http');
+    this.props.getHouses();
+    this.props.getIntegrationByName('nuki');
+
+    this.props.session.dispatcher.addListener(
+      WEBSOCKET_MESSAGE_TYPES.NUKI.NEW_HTTP_DEVICE,
+      this.props.addDiscoveredDevice
+    );
+  }
+
+  componentWillUnmount() {
+    this.props.session.dispatcher.removeListener(
+      WEBSOCKET_MESSAGE_TYPES.NUKI.NEW_HTTP_DEVICE,
+      this.props.addDiscoveredDevice
+    );
+  }
+
+  render(props) {
+    return (
+      <NukiPage user={props.user}>
+        <DiscoverTab {...props} />
+      </NukiPage>
+    );
+  }
+}
```

Comment:
On this page, I don't get any message if the service is not configured

The Zigbee2mqtt integration is a good example: 

![Screenshot 2025-05-08 at 09 18 09](https://github.com/user-attachments/assets/835415b2-709a-456e-8946-41b736572e21)